types/search.ts

```diff
@@ -0,0 +1,12 @@
+export type SearchResult = {
```

Comment:
Usually, on the `types` folder we use interfaces whenever possible. So this one should be an interface.